src/EnergyPlus/WeatherManager.cc

```diff
@@ -4033,7 +4030,7 @@ namespace Weather {
             ASHRAE_C_Coef[8] * (pow_2(pow_2(CosX) - pow_2(SinX)) - pow_2(SinX * CosX * 2.0));
     }
 
-    void CalculateSunDirectionCosines(EnergyPlusData &state,
+    void CalculateSunDirectionCosines(EnergyPlusData const &state,
```

Comment:
[src/EnergyPlus/WeatherManager.cc:4036]:(style),[constParameterReference],Parameter 'state' can be declared as reference to const